.prettierrc

```diff
@@ -0,0 +1,3 @@
+{
+  "singleQuote": true
```

Comment:
This seemed to be the more common pattern so I used this over the default double quote